### PR TITLE
Add article grid page

### PR DESCRIPTION
**Pull request by huangka97**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const query = graphql`
 const IndexPage = ({ data }) => (
   <>
     <Head {...data.site.siteMetadata} />
+    
     <CoverPhoto
```
Comment (ryang72) [inline src/pages/index.tsx]: For xPosition and yPosition can you change them from strings to the enums xPosition and yPosition that are exported in CoverPhoto?

File: src/pages/article_grid.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  const sections=uniq(
+    data.allGoogleSheetRow.edges.map(edge=>
+    capitalizeSection(edge.node.section))
+  )
+
+  const articleGrids=sections.map((section,i) => {
+    const articles = convertEdgesToArticles(
+      data.allGoogleSheetRow.edges
+    ).filter(article => article.section === section.toLowerCase())
+    return <ArticleGrid key={i} heading={section} articles={articles} />
```
Comment (ryang72) [inline src/pages/article_grid.tsx]: add minColumns and maxColumns props.

File: src/pages/article_grid.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import * as React from 'react'
+import { graphql } from 'gatsby'
+import { Article, CoverPhoto, Footer, Head, Headline,ArticleGrid,convertEdgesToArticles } from '@dailybruin/lux'
```
Comment (ryang72) [inline src/pages/article_grid.tsx]: try to use vscode (there should be prettier in atom), the .editorconfig doesn't fix issues like these. Also remove Article and CoverPhoto as they are not used.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+[*]
+charset = utf-8
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (nathunsmitty) [inline .editorconfig]: Add a newline to the end of this file

File: src/pages/article_grid.tsx (hunk 2 above)
Comment (nathunsmitty) [reply src/pages/article_grid.tsx]: actually I think this is fine since min and max columns have default values I think?

Comment (ryang72) [reply src/pages/article_grid.tsx]: then in lux we should make the min and max columns props optional.

Comment (nathunsmitty) [reply src/pages/article_grid.tsx]: If they have default values they are optional – is typescript giving an error here?

Comment (ryang72) [reply src/pages/article_grid.tsx]: at least for me yes.

**Comment by nathunsmitty** [comment]
gonna move some of this logic into lux cause I think it's general enough see dailybruin/lux#1117

**Pull request merged**